moveit_core/collision_detection/src/world.cpp

```diff
@@ -41,11 +41,11 @@
 
 namespace collision_detection
 {
-World::World()
+World::World() : use_detailed_mesh_(false)
```

Comment:
```suggestion
World::World() : use_detailed_mesh_(true)
```
I would expect this to be the default and using convex hulls as an optional speedup/hack. I would also recommend to rename it to `approximate_meshes_using_convex_hull_` (with default `false`)

Reply:
Defaulting to non-convex meshes is going to cause problems because this same code is used by FCL. I think convex hulls must be the default.

Reply:
I agree with @AndyZe. this is an option to use concave mesh. Basically, bullet uses convex-based collision checking. and using concave_mesh_force parameter is an option on the bullet, too.

Reply:
@AndyZe how would bullet being precise (and slower) by default negatively impact FCL-based collision checking?

Reply:
Well, I actually have come around to agree with you, Simon. Since FCL defaults to "detailed meshes", then this setting makes sense:

`use_detailed_mesh_(true)`

Reply:
we need to discuss this with maintainers. @rhaschke @v4hn